.travis.yml

```diff
@@ -23,5 +23,9 @@ install:
 env:
   - TEST_TYPE="pre-commit"
   - TEST_TYPE="tests"
+  - AIIDA_TEST_BACKEND=django
+
+before_script:
+  - ./.travis-data/setup_profiles.sh
```

Comment:
Shouldn't we run this in the "pre-commit" part of the test script instead? The tests themselves use the pytest fixtures which set up a temporary DB + profile.

Reply:
Yes - for now I'm doing it anyway, but yes this could be done only in that case

Reply:
This comment is implicitly fixed in the latest commit where setting up the profile is not done anymore